m2-hotfixes/README.md

```diff
@@ -0,0 +1,20 @@
+# Add custom patches
+
+You can add custom patch files to this directory. Each patch file must use the `.patch` extension, for example the `<patch_name>.patch`.
+There are rules about patches in this directory:
```

Comment:
After all Magento patches are applied, custom patches are applied in alphabetical order from the root of the Magento project as shown in the following example: